src/MainWindow/welcomepagewidget.cpp

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
What's the reason to create a widget within the mainwindow but not by itself like others ?